opsdroid/connector/slack/connector.py

```diff
@@ -119,8 +134,13 @@ async def connect(self):
             _LOGGER.debug(_("Default room is %s."), self.default_target)
 
     async def disconnect(self):
-        """Disconnect from Slack. Only needed when socket_mode_client is used
-        as the Events API uses the aiohttp server"""
+        """Disconnect from Slack.
+
+        Cancels the event queue worker task and disconnects the
+        socket_mode_client if socket mode was enabled."""
```

Comment:
Small nitpick, feel free to ignore
```suggestion
        socket_mode_client if socket mode was enabled.
        
        """
```